frontend/src/assets/images/images.js

```diff
@@ -56,6 +60,8 @@ export {
   iconX,
   // CTJ Logos
   LogoHorizontal,
+  logoHorizontalFooter,
+  logoStackerFooter,
```

Comment:
For clarity, let's put these in alpha order with the lowercase. Also, let's rename them to something else in case we use them somewhere other than the footer. How about "logoXyzOnDark